### PR TITLE
Graph2vol

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
New tool to convert graph object into volumetric image.

# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ ] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: geometry3d/graph2vol.cpp [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+ @brief  Description of the tool...
+
+ @b Usage:   graph2vol [input]
+
+ @b Allowed @b options @b are :
+ 
+
+ @code
+ -h,--help                             Print this help message and exit
+  -v,--inputVertex TEXT:FILE REQUIRED   input file containing the vertex list.
```
Comment (dcoeurjo) [inline geometry3d/graph2vol.cpp:57]: So the vertex list is a sequence of points in Z³ ?

Comment (dcoeurjo) [reply geometry3d/graph2vol.cpp:57]: If so, it should be mentioned that the "embedding" of the vertices is explicitly given

**Review by dcoeurjo — COMMENTED** [review]
Nice PR again ;)

**Comment by kerautret** [comment]
Thanks @dcoeurjo  that done and new option to add margin

**Review by dcoeurjo — APPROVED** [review]

**Pull request merged**